plotly/plotly/plotly.py

```diff
@@ -1493,6 +1493,42 @@ def get_grid(grid_url, raw=False):
         return json_res
 
 
+def create_animations(figure, filename=None, sharing='public'):
```

Comment:
We might want to create an `i` version of this that works in ipython notebooks like `icreate_animations` or else add it as like an output method.